pkgs/development/python-modules/materialx/default.nix

```
@@ -17,13 +17,13 @@
 
 buildPythonPackage rec {
   pname = "materialx";
-  version = "1.39.1";
+  version = "1.38.10";
 
   src = fetchFromGitHub {
     owner = "AcademySoftwareFoundation";
     repo = "MaterialX";
-    rev = "refs/tags/v${version}";
-    hash = "sha256-WzzsY1hOWwJEqT/ZRLIoZDfKNvx1Yf6aFhA3ZcSPx+s=";
+    rev = "v${version}";
```

Why was `refs/tags` removed?

```suggestion
    rev = "refs/tags/v${version}";
```

---

Because this is a direct revert of the update commit

---

Ah, that explains it. Do re-revert this though, it's okay if a revert isn't an exact revert.